ci/docker-compose.yml

```diff
@@ -89,7 +89,7 @@ services:
       KUDU_MASTER: "false"
 
   omniscidb:
-    image: omnisci/core-os-cpu:v5.1.0
+    image: omnisci/core-os-cpu-dev:d7e34d4
```

Comment:
is there not a stable release that can be used?

Reply:
for this PR and for other PRs in progress we need v5.2 that was not released yet. that is why we need the dev version.